### ISSUE TITLE
Image size attribute compatibility with pandoc?

### ISSUE THREAD

**Issue by GandalfSaxe** [typora/typora-issues#1081]
Pandoc supports [link size attributes](http://pandoc.org/MANUAL.html#images). For for example `![](file.jpg){ width=50% }` will work. `pandoc-crossref ` is an indispensable filter for academics that allows referencing of sections, figures, tables, equations etc., supports by the syntax `![](file.jpg){#fig:figlabel}`.

And actually the two can be combined, so the following will work: `![](file.jpg){#fig:figlabel width=50%}`.

It would be really great if Typora could support the image size attribute that is supported by pandoc, instead of only supporting HTML style image sizing `<img src="file.jpg" style="zoom:50%">` because this style cannot be combined with `{#fig:figlabel}`.

In other words, In Typora you currently have to choose between having 1. the correct live preview size or 2. having an image label that can be referenced, but not both.

**Comment by abnerlee** [comment]
merge into https://github.com/typora/typora-issues/issues/249
